.github/workflows/docker-ci.yml

```diff
@@ -143,6 +143,24 @@ jobs:
           docker load --input /tmp/pysages.tar
           docker run --privileged -t pysages bash -c "${SCRIPT}"
 
+  cv-pucker-openmm-unbiased:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: pysages
+          path: /tmp
+      - name: Load and run test
+        run: |
+          docker load --input /tmp/pysages.tar
+          SCRIPT="cd PySAGES/examples/openmm/cv_pucker &&"
+          SCRIPT="${SCRIPT} python unbiased.py" 
+          docker run -t pysages bash -c "cd PySAGES/examples/openmm/cv_pucker && python3 ./unbiased.py"
```

Comment:
Is there a way to pre-record the expected CV and compare with the PySAGES result.

Reply:
It's hard to make it purely deterministic. However, I have scripts to do post-hoc calculation using MDAnalysis and can compare. I guess I can change the unbiased.py, like adding an assert on the deviation between two results?

Reply:
Yeah if you can do that, that works too.
Isn't there a way to set all seeds and then it is determinstic?
Computers are deterministic machines after all.

Whatever works.

Reply:
https://github.com/SSAGESLabs/PySAGES/actions/runs/3298840961/jobs/5441644009

Reply:
cool I'll do that. I said that because I have a vague memory telling me that openmm might not be totally trustworthy on that. I found similar quote in their documentation: 

> 10.4. Determinism
> Whether a simulation is deterministic will depend on what plaform you run on in addition to what settings/methods you use. For instance, as of this writing, using PME on the Reference, OpenCL, and double-precision CUDA will result in deterministic simulations. Single-precision CUDA and CPU platforms are not deterministic in this case. However, none of this behavior is guaranteed in future versions. In many cases it will still result in an identical trajectory. If determinism is a critical for your needs, you should carefully check to ensure that your settings and platform allow for this.

I am just lazy to deal with that lol. But anyways, I'll add it to the script!

Reply:
sure, I understand, anyway of checking it is doing the right thing is fine